src/main/java/com/hearthstone/hearthstone_deck_storage/dto/CardsRO.java

```diff
@@ -0,0 +1,17 @@
+package com.hearthstone.hearthstone_deck_storage.dto;
```

Comment:
package naming must be without underscores. Use hyphen for complex names

Reply:
![image](https://user-images.githubusercontent.com/102020355/206273663-80d24c50-70a1-462e-83a5-d8d56ed6a874.png)

Reply:
i can't use it